k8s/charts/netbox-chart/templates/deployment.yaml

```diff
@@ -36,6 +36,17 @@ spec:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          lifecycle:
+            postStart:
+              exec:
+                command:
+                  - "/bin/sh"
+                  - "-c"
+                  - >
+                    apk add postgresql-dev gcc python3-dev musl-dev zlib-dev jpeg-dev build-base;
```

Comment:
can we build a custom image instead?

Reply:
Tried to revert netbox back to 2.8.5 but there seems to be issues with doing that due to changes to the database. Could restore the database but not sure what other issues that could surface. I had another look to see what exactly is going on and it seems to be to do with the static files. They're supposed to get served from /static but this folder remains empty until you run the collectstatic python task which is also part of the upgrade.sh script so that's why these commands work.

Reply:
do you need to run everything or just the upgrade command?

Reply:
All of it, the first three sort out the pre-reqs for the script